pkg/engine/defaults.go

```diff
@@ -1642,6 +1642,8 @@ func DefaultDetectors() []detectors.Detector {
 		meraki.Scanner{},
 		saladcloudapikey.Scanner{},
 		boxoauth.Scanner{},
+		brandfetchV1.Scanner{},
```

Comment:
Missing Versioner interface implementation. For reference see any detector which has two versions(e.g Github, Gitlab etc)

Reply:
I’ll create an issue to document this better since it can be confusing for most contributors. Even I find the interface smuggling confusing at times, so it’s likely unclear to others as well.

Reply:
> Missing Versioner interface implementation


Can you give me an example of PR where this is implemented.

Reply:
I have made the relevant changes.